lib/bitrange.h

```diff
@@ -65,13 +66,29 @@ class JSONLoader;
 namespace BitRange {
 
 namespace Detail {
+/**
+ * @return find the bit position of the first one bit
+ *
+ * Will return all-ones if no bit is set
+ *
+ * This function is declared constexpr to allow it to be used in tests to optimize code generation.
+ */
+constexpr unsigned ffs(unsigned value) { return sizeof(value) * 8 - absl::countl_zero(value) - 1; }
```

Comment:
@asl I think this comment was on this line (I force-pushed a few times and it seemed to result in losing the comment in github):
> Isn't it countr_zero essentially?

It is as long as value is a power of 2 🙂 
I'm only using it in the case where value is a power of 2, so maybe I will switch. But I'll wait until I've got my build issues sorted out.